jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -154,7 +154,35 @@ jobs:
 ### 2. Split tests
 {: split-tests }
 
-#### a. Split by timing data
+To split tests, pass in a list of tests to the `circleci tests split` command.
+
+The following test splitting options are available:
+
+* Alphabetically by name (by default, if none specified)
+* `--split-by=timings` (recommended)
+* `--split-by=filesize`
+
+#### a. Split by name (default)
+{: #split-by-name }
+
+By default, if you do not specify a method using the `--split-by` flag, `circleci tests split` expects a list of filenames or classnames and splits tests alphabetically by test name. There are a few ways to provide this list:
+
+* Create a text file with test filenames.
+```shell
+circleci tests split test_filenames.txt
+```
+
+* Provide a path to the test files.
+```shell
+circleci tests split < /path/to/items/to/split
+```
+
+* Or pipe a glob of test files.
+```shell
+circleci tests glob "test/**/*.java" | circleci tests split
+```
+
+#### b. Split by timing data
```

Comment:
Should we provide an example in this section using the `--split-by=timings` flag since there are examples in the Name and File size sections?